timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/TestableTimelockCluster.java

```diff
@@ -74,26 +74,30 @@ public TestableTimelockCluster(String baseUri, String defaultClient, String... c
     }
 
     public void waitUntilLeaderIsElected() {
-        waitUntilReadyToServeClients(ImmutableList.of(defaultClient));
+        waitUntilLeaderIsElected(ImmutableList.of());
+    }
+    public void waitUntilLeaderIsElected(List<String> additionalClients) {
+        waitUntilReadyToServeClients(additionalClients);
     }
 
-    public void waitUntilReadyToServeClients(List<String> clients) {
+    private void waitUntilReadyToServeClients(List<String> clients) {
         Awaitility.await()
                 .atMost(60, TimeUnit.SECONDS)
                 .pollInterval(500, TimeUnit.MILLISECONDS)
                 .until(() -> {
                     try {
                         clients.forEach(client -> timelockServiceForClient(client).getFreshTimestamp());
+                        timelockServiceForClient(defaultClient).getFreshTimestamp();
```

Comment:
nit: seems a bit unexpected to me that `waitUntilReadyToServeClients(clients)` also talks to the default client, I'd suggest either having the factories pass `ImmutableList.of(defaultClient)` and [defaultClient + additionalClients] (don't remember efficient syntax offhand), or keep the parameter named `additionalClients` here